doc/widgets.md

```diff
@@ -31,9 +40,12 @@ NineTileBox(
 
 ## SpriteButton
 
-`SpriteButton` is a simple widget that creates a button based on Flame sprites. This can be very useful when trying to create retro looking buttons. For example when creating that look can be more easily achievable by drawing the button in a drawing software, instead of making it directly in Flutter.
+`SpriteButton` is a simple widget that creates a button based on Flame sprites. This can be very
+useful when trying to create non-default looking buttons. For example when creating that look it is
```

Comment:
This sentence doesnt really flow, but I am also not sure what it is meant to say?

Reply:
I agree, quite a mix of my sentence and what was before.
Updated it now to something I think flows better.

Reply:
Yes LGTM.